kbase.yml

```diff
@@ -9,8 +9,8 @@ service-language:
     python
 
 module-version:
-    0.3.13
+    0.4.0
 
 owners:
-    [gaprice, msneddon, jkbaumohl, tgu2]
+    [gaprice, msneddon, jkbaumohl, tgu2, ziming_yang_1]
```

Comment:
Go ahead and remove `msneddon`, he's no longer a kbaser

Reply:
Mike is long gone, please delete it.